Main.sublime-menu

```diff
@@ -1,4 +1,126 @@
 [
+  {
```

Comment:
I would not mind if this file is added,
but I would not add it.

Here is why:
- We have the context menu which let a user choose to run a LSP command (for beginner people starting with LSP)
- We have the command palette which let us access any LSP command (for semi-beginner people starting with LSP)
- We have keybindings which let user choose what LSP command to run (for people who get a feel for how LSP works, and they assigned keybindings to be more productive)

Reply:
I'm convinced we're missing out on users that download this plugin, browse the menu, see nothing, and then uninstall this plugin. Not everyone is aware of a context menu.